src/components/Camera/FocusSquare.tsx

```diff
@@ -1,14 +1,13 @@
 import { View } from "components/styledComponents";
 import _ from "lodash";
-import type { Node } from "react";
 import React from "react";
 import { Animated } from "react-native";
 
 interface Props {
   animatedStyle: Object
 }
 
-const FocusSquare = ( { animatedStyle }: Props ): Node => {
```

Comment:
Functional React components work well with inferred types: https://react-typescript-cheatsheet.netlify.app/docs/basic/getting-started/function_components

Reply:
It sounds like we can remove this Node type from all TS files going forward, yeah?

Reply:
Yes